resdk/resources/base.py

```diff
@@ -118,7 +120,10 @@ def save(self):
         """Save resource to the server."""
         def field_changed(field_name):
             """Check if local field value is different from the server."""
-            return getattr(self, field_name) != self._original_values.get(field_name)
+            field_value = getattr(self, field_name)
+            if isinstance(field_value, ResolweQuery):
+                return False
```

Comment:
@mstajdohar: What do you think about this solution? We cannot compare values if we don't execute query (what we obviously don't want to do)